### PR TITLE
feat: Update http3 lib, Bump version

### PR DESCRIPTION
**Pull request by Sotatek-BinhDo**
## Context
- Investigate xchainpy-thorchain
- Bump version
- Then add whl file into route-proj, or any other proj need

## Change
- Update the version for the http3 lib
- Bump the version
    ecdsa==0.17.0
    requests==2.29.0

## Ticket:
[package dependency and rebuild/update dependencies](https://xdefi-tech.atlassian.net/browse/XDEFI-7806)

### CODE REVIEW

**Review by DJercic — CHANGES_REQUESTED** [review]
we have to fix the version

File: xchainpy/xchainpy_client/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 xchainpy_crypto==0.1.6
-http3==0.6.7
+http3
```
[the remark this answers is not in the document]
Comment (Sotatek-BinhDo) [reply xchainpy/xchainpy_client/requirements.txt]: updated
ref here: 8a02bddc28aa160ad2cb75954805927bb42fcc1c

**Comment by 0xnabil** [comment]
Some dependencies need to be bumped too in order to avoid conflicts at routing project level:
- ecdsa==0.17.0
- requests==2.29.0

**Comment by Sotatek-BinhDo** [comment]
> Some dependencies need to be bumped too in order to avoid conflicts at routing project level:
> 
>     * ecdsa==0.17.0
> 
>     * requests==2.29.0

updated
ref here: cea59758e4150d4b749f8b2bfc86cb48808cf7ec

**Review by 0xnabil — APPROVED** [review]

**Pull request merged**